spec/system/task_spec.rb

```diff
@@ -1,24 +1,26 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
Factoryにassociationを定義するとproject_idを定義する必要が無くなります。
```suggestion
  let(:task) { create(:task) }
```

Reply:
6c1b434 Factoryにassociationを定義し、修正しました。
指摘いただいたように、associationを設定しておけば何度も書く必要がないですね。